README.md

```diff
@@ -1,6 +1,33 @@
 # magmaviz
 
-CThis package contains four different data visualization functions with the magma theme for EDA.
+Exploratory Data Analysis is one of the key steps in a machine learning project. This package aims to make this process easy by providing python functions based on the 'Altair' package to plot four common types of plots with the magma color scheme. To maximize interpretability, the plots have defined color schemes (discrete, diverging, sequential) based on the kind of data they show.
+
+## Functions
+
+The four data visualization functions included in the package are outlined below:
+
+### Boxplot
+
+Returns a boxplot based on the data frame, a numerical feature to view the distribution of and a categorical feature to bucket data into categories. Additionally, there is a boolean option to facet the boxplots into separate charts.
+
+### Correlation Plot
+
+Returns a correlation plot based on the numerical features present in the data frame. While the default plot would use circle shapes, an auxiliary input provides the flexibility to switch to square shapes. Additionally, it will print the correlated numerical feature pairs along with their correlation values.
+
+### Histogram
+
+Returns a histogram based on the data frame and a categorical feature to plot on the x-axis. The y-axis will display the count of records.
+
+### Scatterplot
+
+Returns a scatterplot based on the data frame and two numerical feature names passed as the required inputs. An auxiliary input provides the flexibility to color-code the data points based on a categorical feature.
+
+### Fit within the Python ecosystem
+
+Our package will build onto the existing features of 'Altair' using the magma color scheme. It serves as an automated plotter and is a higher level implementation of it. Essentially it circumvents the need to code every single detail and allows the user to focus on the output. We came across two packages that have a similar line of thought:
+
+https://pypi.org/project/deneb/ (Altair) - uses the same base as this package
```

Comment:
Should we have these as hyperlinks?

Reply:
I reckon that may be more useful.

Reply:
Makes sense. Adding hyperlinks with package names.

Reply:
Could you please update the histogram function to say "Returns a histogram based on the data frame and a categorical feature to plot on the x-axis. The y-axis will display the user-specified aggregation function."